tests/demo_data/dementia_v_0_1/CDEsMetadata.json

```diff
@@ -1551,7 +1552,7 @@
           "isCategorical": true,
           "code": "alzheimerbroadcategory",
           "sql_type": "text",
-          "description": "There will be two broad categories taken into account. Alzheimer s disease (AD) in which the diagnostic is 100% certain and <Other> comprising the rest of Alzheimer s related categories. The <Other> category refers to Alzheime s related diagnosis which origin can be traced to other pathology eg. vascular. In this category MCI diagnosis can also be found. In summary  all Alzheimer s related diagnosis that are not pure.",
+          "description": "There will be two broad categories taken into account. Alzheimer s disease (AD) in which the diagnostic is 100% certain and <Other> comprising the rest of Alzheimer s related categories. The <Other> category refers to Alzheime s related diagnosis which origin can be traced to other data_model eg. vascular. In this category MCI diagnosis can also be found. In summary  all Alzheimer s related diagnosis that are not pure.",
```

Comment:
This is MIP specific. Don't change to data model.

Will be removed anyway in the future but... whatever...